alan/coo-1005-docs-update-vendor-management (#19)
IdentitySquare/CookieChimp-Docs: docs/features/vendor-management.mdx

---
title: "Vendor Management"
description: "Organize and manage storage items used your website."
---

## Categories

> Categories are used to organize your cookies into groups. You can create as many categories as you need. The default categories are:

Reviewer: To organise vendors

Reviewer: They can add and edit existing ones we have setup as default.

- Essential
- Analytics
- Marketing
- Personalization

> Users will be consenting to these groups of cookies. Each group will have a toggle switch to allow users to opt-in or opt-out of the category.

Reviewer: We can delete this line. 

```suggestion
```

<img src="/images/vendor-management/categories.png" alt="Categories" />

A category becomes active when it contains an active service or storage items. Only active categories are shown to visitors in the banner.

## Vendors

Storage items set by a vendor/service can be grouped together.
Services must be associated with a category and each service will have a toggle inside its category section to allow users to opt-in or opt-out of the service.

> You can then use the services attribute to [block scripts](/docs/block-scripts-cookies).

Reviewer: Block scripts until consent

CookieChimp offers a vendor library of popular services that contain storage items set by the service.
Simply select the ones used in your site and add.

<img src="/images/vendor-management/vendor-library.png" alt="Vendor Library" />

> If you are using a service that is not in the vendor library, create the service yourself and add any storage items used by the service.

Reviewer: ```suggestion
If you are using a service that is not in our vendor library, you can create a service manually.
```
